chained-actions/env-init.sh

```diff
@@ -0,0 +1 @@
+ssh root@host01 "yum -y install wget tree"
```

Comment:
Do we really need this?

Reply:
probably not anymore.  a relic of the template this was based on.